lightning/src/ln/chan_utils.rs

```diff
@@ -25,6 +25,25 @@ use secp256k1;
 pub(super) const HTLC_SUCCESS_TX_WEIGHT: u64 = 703;
 pub(super) const HTLC_TIMEOUT_TX_WEIGHT: u64 = 663;
 
+#[derive(PartialEq)]
+pub(super) enum HTLCType {
+	AcceptedHTLC,
+	OfferedHTLC,
+	NoneHTLC,
+}
+
+impl HTLCType {
+	pub(super) fn weight_to_htlctype(witness_script_len: usize) ->  HTLCType {
```

Comment:
Could use some at least basic docs, like "checks if a given tx weight is potentially is pre-signed HTLC transaction"